frontend/src/data/loading/DataFetcher.ts

```diff
@@ -96,17 +96,27 @@ export class ApiDataFetcher implements DataFetcher {
       datasetId.startsWith("acs_2010_population")
     ) {
       result = result.map((row: any) => {
-        return { ...row, population: Number(row["population"]) };
+        return {
```

Comment:
Just making sure I understand that we can still get rid of these when everything is done on the backend?

Reply:
Yes I believe so; I've outlined what i think the overall solution should be here: #1662